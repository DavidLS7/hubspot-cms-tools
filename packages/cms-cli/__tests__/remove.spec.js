/*global banjo */

describe('Remove command', () => {
  it('works with help positional', async () =>
    expect(await banjo(['remove', 'help'])).toMatchSnapshot());

  it('works with help option', async () =>
    expect(await banjo(['remove', '--help'])).toMatchSnapshot());
});
