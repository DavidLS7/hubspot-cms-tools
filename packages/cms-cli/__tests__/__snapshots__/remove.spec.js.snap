// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remove command works with help option 1`] = `
"banjo remove <path>

Delete a file or folder from HubSpot

Positionals:
  path  Remote hubspot path                                  [string] [required]

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;

exports[`Remove command works with help positional 1`] = `
"banjo remove <path>

Delete a file or folder from HubSpot

Positionals:
  path  Remote hubspot path                                  [string] [required]

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;
